### PR TITLE
[scudo] Add the record of number of attempted page release

### PR DESCRIPTION
**Pull request by ChiaHungDuan**
This also removes the `RangesReleased` which doesn't give much insight to whether we should adjust the heuristic of doing page release.

### CODE REVIEW

File: compiler-rt/lib/scudo/standalone/primary32.h [hunk 1]
```diff
@@ -880,14 +880,14 @@ template <typename Config> class SizeClassAllocator32 {
...
-                "inuse: %6zu avail: %6zu releases: %6zu last released: %6zuK "
-                "latest pushed bytes: %6zuK\n",
-                ClassId, getSizeByClassId(ClassId), Sci->AllocatedUser >> 10,
-                Sci->FreeListInfo.PoppedBlocks, Sci->FreeListInfo.PushedBlocks,
-                InUse, AvailableChunks, Sci->ReleaseInfo.RangesReleased,
-                Sci->ReleaseInfo.LastReleasedBytes >> 10,
-                PushedBytesDelta >> 10);
+    Str->append(
+        "  %02zu (%6zu): mapped: %6zuK popped: %7zu pushed: %7zu "
+        "inuse: %6zu avail: %6zu release count: %6zu last released: %6zuK "
```
Comment (cferris1000) [inline compiler-rt/lib/scudo/standalone/primary32.h:885]: Should this be change to releases attempted?

File: compiler-rt/lib/scudo/standalone/primary32.h [hunk 2]
```diff
@@ -387,7 +387,7 @@ template <typename Config> class SizeClassAllocator32 {
 
   struct ReleaseToOsInfo {
     uptr BytesInFreeListAtLastCheckpoint;
-    uptr RangesReleased;
+    uptr NumReleaseAttempted;
```
Comment (cferris1000) [inline compiler-rt/lib/scudo/standalone/primary32.h:390]: This is such a nit, but could this be NumReleasesAttempted?

File: compiler-rt/lib/scudo/standalone/primary32.h (hunk 1 above)
Comment (ChiaHungDuan) [reply compiler-rt/lib/scudo/standalone/primary32.h]: Done

File: compiler-rt/lib/scudo/standalone/primary32.h (hunk 2 above)
Comment (ChiaHungDuan) [reply compiler-rt/lib/scudo/standalone/primary32.h]: Done

File: compiler-rt/lib/scudo/standalone/primary32.h [hunk 3]
```diff
@@ -880,14 +880,14 @@ template <typename Config> class SizeClassAllocator32 {
...
-                "inuse: %6zu avail: %6zu releases: %6zu last released: %6zuK "
-                "latest pushed bytes: %6zuK\n",
-                ClassId, getSizeByClassId(ClassId), Sci->AllocatedUser >> 10,
-                Sci->FreeListInfo.PoppedBlocks, Sci->FreeListInfo.PushedBlocks,
-                InUse, AvailableChunks, Sci->ReleaseInfo.RangesReleased,
-                Sci->ReleaseInfo.LastReleasedBytes >> 10,
-                PushedBytesDelta >> 10);
+    Str->append(
+        "  %02zu (%6zu): mapped: %6zuK popped: %7zu pushed: %7zu "
+        "inuse: %6zu avail: %6zu release attempted: %6zu last released: %6zuK "
```
Comment (cferris1000) [inline compiler-rt/lib/scudo/standalone/primary32.h:885]: releases attempted

File: compiler-rt/lib/scudo/standalone/primary64.h [hunk 4]
```diff
@@ -530,7 +530,7 @@ template <typename Config> class SizeClassAllocator64 {
 
   struct ReleaseToOsInfo {
     uptr BytesInFreeListAtLastCheckpoint;
-    uptr RangesReleased;
+    uptr NumReleaseAttempted;
```
Comment (cferris1000) [inline compiler-rt/lib/scudo/standalone/primary64.h:533]: Can you change the name to NumReleasesAttempted here too?

File: compiler-rt/lib/scudo/standalone/primary64.h [hunk 5]
```diff
@@ -1144,11 +1144,12 @@ template <typename Config> class SizeClassAllocator64 {
     Str->append(
         "%s %02zu (%6zu): mapped: %6zuK popped: %7zu pushed: %7zu "
         "inuse: %6zu total: %6zu releases: %6zu last "
-        "released: %6zuK latest pushed bytes: %6zuK region: 0x%zx (0x%zx)\n",
+        "release attempted: %6zuK latest pushed bytes: %6zuK region: 0x%zx "
```
Comment (cferris1000) [inline compiler-rt/lib/scudo/standalone/primary64.h:1147]: releases attempted

File: compiler-rt/lib/scudo/standalone/primary32.h (hunk 3 above)
Comment (ChiaHungDuan) [reply compiler-rt/lib/scudo/standalone/primary32.h]: Done

File: compiler-rt/lib/scudo/standalone/primary64.h (hunk 4 above)
Comment (ChiaHungDuan) [reply compiler-rt/lib/scudo/standalone/primary64.h]: Done.

File: compiler-rt/lib/scudo/standalone/primary64.h (hunk 5 above)
Comment (ChiaHungDuan) [reply compiler-rt/lib/scudo/standalone/primary64.h]: Done.

**Review by ChiaHungDuan — COMMENTED** [review]
Sorry for not paying attention to all the places that needed to change

**Review by cferris1000 — APPROVED** [review]
LGTM

**Pull request merged**